tdms/src/iterator.cpp

```diff
@@ -7027,74 +6959,40 @@ double linearRamp(double t, double period, double rampwidth) {
     return t / (period * rampwidth);
 }
 
-//i_l is the index into the fdtd grid which is the first non-pml cell in the i direction
-//i_u is the index into the fdtd grid which is the last non-pml cell in the i direction
-double checkPhasorConvergence(ElectricField &E, ElectricField &E_copy, ElectricSplitField &E_s,
-                              int i_l, int i_u, int j_l, int j_u, int k_l, int k_u, int n,
-                              double omega, double dt, int Nt) {
+double checkPhasorConvergence(ElectricField &E, ElectricField &E_copy, ElectricSplitField &E_s) {

```

Comment:
`E_s` doesn't seem to be used in this function, so get rid of it from the signature?

Reply:
weird. it was in the [original signature](https://github.com/t-young31/ifdtd/blob/5d6338f5587368814a0874aac6f75e15f7e86b5b/iterater/iterater_OMP_PSTD.cpp#L7120), but yeah, unused